device.go

```diff
@@ -89,6 +90,13 @@ type DeviceInfo struct {
 
 	// id of the device group.
 	GID uint32
+
+	// Hotplugged is used to store device state indicating if the
+	// device was hotplugged.
+	Hotplugged bool
+
+	// ID passed on the hypervisor command line.
```

Comment:
This comment doesn't seem quite correct given you use this value to construct a new string value for the hypervisor?

Reply:
@jodh-intel I have updated the comment.